<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>341</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>563</width>
    <height>341</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>563</width>
    <height>341</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PyVideo</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.ico</normaloff>icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QVideoWidget" name="wgt_player" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>501</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:7px;
background-color: rgb(197, 197, 197);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_play_pause">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>292</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
image: url(:/imge/暂停.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="sld_duration">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>275</y>
      <width>501</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
        height: 2.5px;
margin: 0px 0;  
        background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 rgb(203, 203, 203), stop: 1.0 rgb(203, 203, 203));
}
QSlider::handle:horizontal {
        width: 9px;
		height: 5px;
	background-color: rgb(44, 44, 44);
        margin: -5px 0px -5px 0px;
     border-radius:4px;
}
  
     QSlider::add-page:Horizontal
     {     
        
        height:4px;
     }
     QSlider::sub-page:Horizontal 
    {
        
	background-color: rgb(0, 170, 255);
        height:4px;
     }\
    QSlider::groove:Horizontal
    {\
        background:transparent;
        height:6px;
    }\
    QSlider::handle:Horizontal 
    {
        height: 30px;
        width:8px;
        margin: -8 0px; 
    }
</string>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lab_duration">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>292</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>--:--</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_select">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>290</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:rgb(153, 153, 153);
color: rgb(255, 255, 255);
border-radius:4px;</string>
    </property>
    <property name="text">
     <string>选择视频</string>
    </property>
   </widget>
   <widget class="QSlider" name="volumeSlider">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>290</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
        height: 2.5px;
margin: 0px 0;  
        background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 rgb(203, 203, 203), stop: 1.0 rgb(203, 203, 203));
}
QSlider::handle:horizontal {
        width: 9px;
		height: 5px;
	background-color: rgb(44, 44, 44);
        margin: -5px 0px -5px 0px;
     border-radius:4px;
}
  
     QSlider::add-page:Horizontal
     {     
        
        height:4px;
     }
     QSlider::sub-page:Horizontal 
    {
        
	background-color: rgb(0, 170, 255);
        height:4px;
     }\
    QSlider::groove:Horizontal
    {\
        background:transparent;
        height:6px;
    }\
    QSlider::handle:Horizontal 
    {
        height: 30px;
        width:8px;
        margin: -8 0px; 
    }
</string>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>8</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="imge">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>290</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/imge/volume.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">PyQt5.QtMultimediaWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
